statistician/Dockerfile

```diff
@@ -10,17 +10,22 @@ COPY requirements.txt version.txt /conf/
 RUN cat /conf/version.txt \
     && env-build-tool new /conf/requirements.txt ${py_env_path} /wheels
 
+# Below is the actual image that does the running
 FROM opendatacube/geobase:runner
 
-ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBIAN_FRONTEND=noninteractive \
+    PATH="/env/bin:${PATH}" \
+    LC_ALL=C.UTF-8 \
+    LANG=C.UTF-8
 
 COPY --from=env_builder /env /env
 
+# Add in the dask configuration
+COPY distributed.yaml /root/.config/dask/distributed.yaml 
```

Comment:
When building docker image I would put it into `/etc/dask/` folder instead, see here:

`https://docs.dask.org/en/latest/configuration.html`

This would apply to any user, we might not stay running as root forever.

Also it should probably go above environment copy step, we do not expect it to be tweaked very frequently.